src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/CaseCommentRequest.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.probation.courtcaseservice.controller.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.justice.probation.courtcaseservice.jpa.entity.CaseCommentEntity;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CaseCommentRequest {
+    @NotBlank
+    private final String caseId;
+    @NotBlank
+    private final String comment;
```

Comment:
I think we'll also need a field for the Author 🤔 We've got the created by fields which are populated from the user token but that only holds the username which often isn't very readable. This must be in scope in `prepare-a-case` for it to render it on the front end so we should be able to pass it through.

Reply:
I thought created by is what we are interested in. Will update to receive the author.